### PR TITLE
Boxplot Method - Climatic Menus.

### PR DESCRIPTION
**Pull request by aduut**
Boxplot Method with all parameters.

### CODE REVIEW

**Comment by maxwellfundi** [comment]
Hi Abib, 
Kindly change the naming conventions for checkboxes for the dlgBoxplotMethod - change Checkbox1 to the appropriate name

File: instat/dlgBoxplotMethod.vb [hunk 1]
```diff
@@ -6,4 +6,54 @@ Public Class dlgBoxplotMethod
         ucrBase.clsRsyntax.iCallType = 0
     End Sub
 
+    Private Sub CheckBox1_CheckedChanged(sender As Object, e As EventArgs) Handles chkConvert.CheckedChanged
+        If chkConvert.Checked Then
```
Comment (volloholic) [inline instat/dlgBoxplotMethod.vb]: This is the comment that Maxwell made the code will run fine as it is but it is good practice to name this sub chkConvert_CheckedChanged for consistency.

**Comment by volloholic** [comment]
Hi Abib why are you using textchanged? We will need to discuss this before I can accept the merge.

**Comment by maxwellfundi** [comment]
This is now redone and ready to be closed
thanks

**Pull request closed without merging**